src/accumulator/mod.rs

```diff
@@ -1,3 +1,67 @@
+//! This module is the core of the library. It contains all the basic data structures to use Utreexo,
+//! such as accumulator, and proofs. Even though the algorithms are equal, you may use different
+//! data structures to represent the accumulator, each with its own advantages and disadvantages.
+//!
+//! # [Stump]
+//! A Stump is a basic data structure used in Utreexo. It only holds the roots and the number of leaves
+//! in the accumulator. This is useful to create lightweight nodes, the still validates, but is more compact,
```

Comment:
`s/This is useful to create lightweight nodes, the still validates, but is more compact, perfect to clients running on low-power devices./This is useful for creating lightweight nodes that still perform validation, making it suitable for clients on low-power devices.`